python/ray/data/_internal/stats.py

```diff
@@ -339,61 +346,131 @@ def _check_cluster_stats_actor():
         _stats_actor_cluster_id = current_cluster_id
 
 
-def update_stats_actor_metrics(
-    op_metrics: List[OpRuntimeMetrics],
-    tags_list: List[Dict[str, str]],
-    state: Dict[str, Any],
-):
-    global _stats_actor
-    _check_cluster_stats_actor()
+# After this many iterations of inactivity,
+# StatsActorManager._iter_update_thread will close itself.
+STATS_ACTOR_UPDATE_THREAD_INACTIVITY_LIMIT = 5
```

Comment:
nit, some file-level constants can be defined in the class now, if they are internal to the class.